### PR TITLE
Ft api postgres with db

### PR DESCRIPTION
**Pull request by dondrzzy**
Hey team, please take a look at this API and see if its okay or close to okay. Feedback will be very much appreciated, I implemented the database and added some MVC logic

### CODE REVIEW

**Comment by RrNn** [comment]
All checks have failed.

**Comment by RrNn** [comment]
@dondrzzy

**Comment by Ashaba** [comment]
Please resolve conflicts to the base branch and proceed to merging

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - postgresql
+install:
+  - pip3 install -r requirements.txt
+  - pip3 install pytest pytest-cov
+  - pip3 install coveralls
+before_script:
+  - psql -c "CREATE DATABASE tbc_db;" -U postgres
+  - psql -c "CREATE USER tbc_db WITH PASSWORD '12345678';" -U postgres
+script:
+  - pytest test --cov=test/
```
Comment (Ashaba) [inline .travis.yml]: Good work so far. However, this line `pytest test --cov=test` will return your coverage for functionality in the package test which I think is not what you might be looking for. I would recommend you consider reporting the coverage for the entire application. in this case you would replace `test` with `app`

Comment (dondrzzy) [reply .travis.yml]: Yes, that's so true. I set coverage to the test directory when i was still trying to figure out what coveralls are.

**Comment by rogerokello** [comment]
Good work on the persistent features

**Pull request merged**